cortex-plugins-core.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE.md", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5"
-  s.add_dependency "react_on_rails", "~> 8"
+  s.add_dependency "react_on_rails", "~> 9"
```

Comment:
I don't think this dependency will be necessary since these components wont be mounting on their own.

Reply:
Good catch! I've removed this superfluous dependency from the `gemspec` and `package.json`